client/src/components/Header.js

```diff
@@ -3,9 +3,14 @@ import { BrowserRouter as Router, useLocation } from "react-router-dom";
 export default function Header({ editLearningObjectives, logout, back }) {
   let location = useLocation();
 
-  const logoLink = location.pathname.includes("skills")
-    ? "/skills"
-    : "/MentorsView";
+  const [logoLink, setLogoLink] = useState("");
+  useEffect(() => {
+    if (location.pathname.includes("skills")) {
+      setLogoLink("/skills");
+    } else if (location.pathname.includes("Mentors")) {
+      setLogoLink("/MentorsView");
+    }
+  }, []);
```

Comment:
How come this had to be reverted?

Reply:
I am Sorry I should have explained that; so what happened is the header where the navbar (logo) is used by all pages including the landing page and the sign up form and the logo became clickable to the /MentorView path by default, so the only way I could do it is to assign it using useState to be able to make a default value as "", and the useEffect is because the page had too many renders error?!